src/lib/stores/drips/utils/__test__/mockObjects.ts

```diff
@@ -0,0 +1,21 @@
+import type { DripHistoryEvent } from '$lib/stores/workstreams';
```

Comment:
I find the word "mock" used in this context a bit confusing. I normally associate mock objects with objects that can be made to change their behaviour within a test or measure whether any of their attributes get called.

What `mockMoney` and `mockDripHistoryEvent` do is wrap the object creation into a function, effectively making it shorter to write out. I'd move them into the test and call them something like `createDai` and `createDripHistoryEvent` or something like that.